examples/hybrid/driver.jl

```diff
@@ -19,6 +19,7 @@ idealized_insolation = parsed_args["idealized_insolation"]
 idealized_clouds = parsed_args["idealized_clouds"]
 vert_diff = parsed_args["vert_diff"]
 surface_scheme = parsed_args["surface_scheme"]
+C_E = parsed_args["C_E"]
```

Comment:
Let's put this in `model_spec` if we're planning on merging this (`C_E = FT(C_E)` below adds a closure over a global variable).

Reply:
Shouldn't `C_E` be similar to things like `kappa_4` though? Or do we plan to move all these things in `model_spec`?

Reply:
We're planning to move all of these things into `model_spec`, in order to remove the global closures.

Reply:
Oh ok, thanks!

Reply:
Changed, @charleskawczynski could you take a look again?